prowler/compliance/aws/aws_account_security_onboarding_aws.json

```diff
@@ -0,0 +1,445 @@
+        ]
+      },
+      {
+        "Id": "Alerting",
+        "Name": "Alerting",
+        "Description": "Enable alerts in your AWS Account.",
+        "Attributes": [
+          {
+            "Section": "Guardutty",
+            "Name": "GuardDuty"
+          },
+          {
+            "Section": "Cloudtrail",
+            "Name": "Cloudtrail"
+          },
+          {
+            "Section": "Cost-Anomaly",
+            "Name": "Cost-Anomaly"
+          },
+          {
+            "Section": "SecurityHub",
+            "Name": "SecurityHub"
+          },
+          {
+            "Section": "WAF",
+            "Name": "WAF"
+          },
+          {
+            "Section": "Shield",
+            "Name": "Shield"
+          },
+          {
+            "Section": "R53-DNS-Resolver",
+            "Name": "R53-DNS-Resolver"
+          },
+          {
+            "Section": "Vulnerability-Scanning",
+            "Name": "Vulnerability-Scanning"
+          },
+          {
+            "Section": "Prod",
+            "Name": "Prod"
+          },
+          {
+            "Section": "QA",
+            "Name": "QA"
+          }
+        ],
+        "Checks": [
+          "guardduty_is_enabled",
+          "guardduty_no_high_severity_findings",
+          "route53_domains_privacy_protection_enabled",
+          "securityhub_enabled",
+          "elbv2_waf_acl_attached",
+          "apigateway_restapi_waf_acl_attached",
+          "shield_advanced_protection_in_associated_elastic_ips",
+          "shield_advanced_protection_in_classic_load_balancers",
+          "shield_advanced_protection_in_cloudfront_distributions",
+          "shield_advanced_protection_in_global_accelerators",
+          "shield_advanced_protection_in_internet_facing_load_balancers",
+          "shield_advanced_protection_in_route53_hosted_zones",
+          "ecr_repositories_scan_images_on_push_enabled",
+          "ecr_repositories_scan_vulnerabilities_in_latest_image"
+        ]
+      },
+      {
+        "Id": "Budget-Alarms",
+        "Name": "Budget-Alarms",
+        "Description": "Budget Alarms for your AWS Account.",
+        "Attributes": [
+          {
+            "Section": "Prod",
+            "Name": "Prod"
+          },
+          {
+            "Section": "QA",
+            "Name": "QA"
+          }
+        ],
+        "Checks": []
+      },
+      {
+        "Id": "IAM-Access-Analyzer",
+        "Name": "IAM access analyzer",
+        "Description": "IAM access analyzer in your AWS Account.",
+        "Attributes": [
+          {
+            "Section": "Analyzer-Regions",
+            "Name": "Create analyzers in each active regions"
+          },
+          {
+            "Section": "Verify-that-events-are-present-in-SecurityHub-aggregated-view",
+            "Name": "Verify that events are present in SecurityHub aggregated view"
+          }
+        ],
+        "Checks": [
+          "securityhub_enabled"
```

Comment:
can you add these ones?
<img width="276" alt="image" src="https://github.com/prowler-cloud/prowler/assets/38561120/e2e69f99-7b61-4f75-bc5f-ea0190845aa9">

Reply:
Done!